### PR TITLE
fix: update colors of OPTION, TRACE and HEAD methods

### PR DESCRIPTION
**Pull request by gaetanmaisse**
**Issue**

NA

**Description**

 - Update colors of OPTION, TRACE, and HEAD methods
 - Add a manual approval step to not trigger Chromatic each time

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://6183b02d73381a003a3be1a6-eqbuhrhlpu.chromatic.com)

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@2.6.1-add-missing-method-colors-979d13a
# or
yarn add @gravitee/ui-particles-angular@2.6.1-add-missing-method-colors-979d13a
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@2.6.1-add-missing-method-colors-979d13a
# or
yarn add @gravitee/ui-policy-studio-angular@2.6.1-add-missing-method-colors-979d13a
```

File: .circleci/config.yml [hunk 1]
```diff
@@ -296,12 +296,16 @@ workflows:
           package_name: ui-particles-angular
           requires:
             - Build Angular
+      - trigger-chromatic:
+          type: approval
+          requires:
+            - Build Angular Storybook
```
Comment (ThibaudAV) [inline .circleci/config.yml:302]: 💸  ?

**Review by ThibaudAV — APPROVED** [review]

File: ui-particles-angular/projects/ui-particles-angular/src/scss/gio-mat-palettes.scss [hunk 2]
```diff
@@ -163,12 +163,18 @@ $mat-method-palette: mat.define-palette(
     put: #fbf2f2,
     get: #ebf7ff,
     delete: #fff5f4,
+    option: #f4f6ff,
```
Comment (ThibaudAV) [inline ui-particles-angular/projects/ui-particles-angular/src/scss/gio-mat-palettes.scss:166]: to be added here too : https://github.com/gravitee-io/gravitee-ui-particles/blob/c14f80ce33f5f6bb87a1dd2ec6dbf8bcf061ff15/ui-particles-angular/projects/ui-particles-angular/src/scss/theme/color/gio-palettes.stories.scss#L53

Comment (ThibaudAV) [reply ui-particles-angular/projects/ui-particles-angular/src/scss/gio-mat-palettes.scss:166]: to have color inside storybook

File: .circleci/config.yml (hunk 1 above)
Comment (gaetanmaisse) [reply .circleci/config.yml:302]: Yes

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@2.6.1-add-missing-method-colors-6709bc5
# or
yarn add @gravitee/ui-policy-studio-angular@2.6.1-add-missing-method-colors-6709bc5
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@2.6.1-add-missing-method-colors-6709bc5
# or
yarn add @gravitee/ui-particles-angular@2.6.1-add-missing-method-colors-6709bc5
```

**Pull request merged**